_posts/2019-12-16-Image-caching-with-FFImageloading.md

```diff
@@ -113,21 +114,18 @@ protected override void OnLowMemory(LowMemoryEventArgs e)
 }
 ```
 
- You can also clear cache on the disk.
+ You can also clear cache on the disk:
 ``` c#
 FFImageLoading.ImageService.Instance.InvalidateDiskCacheAsync();
 ```
 
-## The Best usage of CachedImage
- `CachedImage` works best when it is used in `CollectionView`, because `CollectionView` unloads a view that is out of sight and reloads when it comes back to the sight. Once image source is loaded, it shows really quick.
+## Best use of CachedImage
+ `CachedImage` works best when it is used in `CollectionView`, because `CollectionView` unloads (Question: Is unloads the right word?) a view that is out of sight and reloads when it comes back to the sight. Once the image source is loaded, it appears quickly.
```

Comment:
well, I think it sounds ok to me, like `load`, `unload`, and `reload` a view. Or realize and unrealize a view. Does it sounds weird to you?